package.json

```diff
@@ -285,8 +285,7 @@
     "@types/jest-axe/axe-core": "4.2.1",
     "@types/react": "16.14.23",
     "@types/react-dom": "16.9.14",
-    "eslint": "7.25.0",
-    "copy-to-clipboard": "3.2.0"
```

Comment:
needed to remove `copy-to-clipboard` from `resolutions` 

- syncpack was erroring out on this withoun `node_modules` present
- deduped yarn.lock
- following single version policy to have all 3rd party packages present in root package.json